assets/stylesheets/index.scss

```diff
@@ -9,10 +9,12 @@
 // Components
```

Comment:
As per our convention, we can remove `_` prefix while importing 
![Screen Shot 2021-05-21 at 5 14 18 PM](https://user-images.githubusercontent.com/6076762/119121779-0513bc80-ba58-11eb-8cbf-37012c9c2257.png)

Reply:
✅  removed in [27d4648](https://github.com/carryall/go-google-scraper-challenge/pull/57/commits/27d464824352f9fb96d9d4f2dde4f9adffa91100)